src/@types/content-security-policy-parser/index.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module 'content-security-policy-parser' {
+  function fn(
+    policy: string,
+  ): {
+    [directive: string]: string;
+  };
+  export = fn;
```

Comment:
We should probably have a comment in this file to explain that the module exposes a default anonymous function.